packages/messages/src/domains/messages.js

```diff
@@ -6,8 +6,9 @@ const connection = require('../drivers/mongodb/connection');
 
 const Message = new Schema({
   chatId: { type: ObjectId, require: true, ref: ChatModel },
-  text: { type: String, lowercase: true, trim: true },
```

Comment:
We don't need to lower case the message text, this would be a weird user experience